packages/starlight/components/SidebarSublist.astro

```diff
@@ -79,6 +85,8 @@ interface Props {
 		align-items: center;
 		justify-content: space-between;
 		padding-inline: var(--sl-sidebar-item-padding-inline);
+		padding-block-end: 0.3em;
+		line-height: 1.5;
```

Comment:
Nice — thanks @kevinzunigacuellar!

Played around with it a bit and this looks closest to the current styles to me. Preserves an overall spacing of 1.8 (for single-line items), tightens up the line-height for items that wrap, and distributes the extra padding to the top and bottom.

```suggestion
		padding-block: 0.2em;
		line-height: 1.4;
```

| Before either of our changes | After applying this suggestion |
|---|---|
| <img width="293" alt="image" src="https://github.com/withastro/starlight/assets/357379/461218b7-efce-4b91-9d95-f7bc00c50418"> | <img width="293" alt="image" src="https://github.com/withastro/starlight/assets/357379/30ae7c8f-e991-45ca-8fc8-545bdf93f4de"> |

Reply:
Thanks for taking the time to tweak this. Looks perfect on my end